libs/community/tests/unit_tests/vectorstores/test_sqlserver.py

```diff
@@ -14,6 +16,18 @@
 )
 
 _CONNECTION_STRING = str(os.environ.get("TEST_AZURESQLSERVER_CONNECTION_STRING"))
+_CONNECTION_STRING_WITH_UID_AND_PWD = str(
+    os.environ.get("TEST_AZURESQLSERVER_CONNECTION_STRING_WITH_UID")
+)
+_CONNECTION_STRING_WITH_TRUSTED_CONNECTION = str(
+    os.environ.get("TEST_AZURESQLSERVER_TRUSTED_CONNECTION")
+)
+_ENTRA_ID_CONNECTION_STRING_NO_PARAMS = str(
+    os.environ.get("TEST_ENTRA_ID_CONNECTION_STRING_NO_PARAMS")
+)
+_ENTRA_ID_CONNECTION_STRING_TRUSTED_CONNECTION_NO = str(
```

Comment:
How do you set the environment variable values?

Reply:
This is expected to be provided as part of the test resources, i.e. preset in the environment where the test will run. However, since we do not have test resources set up yet, for running this locally, it has to be set manually beforehand.

Reply:
can you please add to the description, or somewhere in the test?